src/System.Security.SecureString/tests/SecureStringTests.cs

```diff
@@ -292,18 +292,24 @@ public static void SecureString_SetAt()
 
         using (SecureString testString = CreateSecureString("test"))
         {
-            Assert.Throws<ArgumentOutOfRangeException>(() => testString.RemoveAt(-1));
-            Assert.Throws<ArgumentOutOfRangeException>(() => testString.RemoveAt(testString.Length));
-            Assert.Throws<ArgumentOutOfRangeException>(() => testString.RemoveAt(testString.Length + 1));
+            Assert.Throws<ArgumentOutOfRangeException>(() => testString.SetAt(-1, 'a'));
```

Comment:
The other assets exist in the test for those methods; I think these were copy and paste errors from those.